news/2025/2025-01-23-osumania-7k-world-cup-2025-round-of-32-recap.md

```diff
@@ -0,0 +1,186 @@
+
+France's last pick ended up being [II-L - VANGUARD-3](https://osu.ppy.sh/beatmapsets/2307703#mania/4935651) (RC4), letting ::{ flag=FR }:: [Whirst](https://osu.ppy.sh/users/5350761) do the heavy lifting. Unfortunately, ::{ flag=FR }:: Mokett's combo wasn't able to match up to their peer's. Even with Macau's loss of accuracy on the very last segment, they were still able to hold up with a meager **153 point advantage** — yet another minute difference in scores! As their last pick, Macau chose [BPM15Q - Kyo You Mu](https://osu.ppy.sh/beatmapsets/2307429#mania/4934922) (HB2). Thanks to ::{ flag=MO }:: [Randumbs](https://osu.ppy.sh/users/27657112)'s impressive 974,493 play and both of the players' efforts, it was time to break the tie that this last map produced.
+
+Saying that the tiebreaker was hard, would be an understamenet. Despite fighting tooth and nail, France managed to stay on top with their lead in ratio. While France may be going into the next round facing ::{ flag=CN }:: China, Macau's surely left a mark with their impressive performance this match, having to face ::{ flag=MX }:: Mexico in the round to come.
+
+### ::{ flag=VE }:: Venezuela vs. ::{ flag=CA }:: Canada
+
+<style>
+    .news-chat-quote__avatar {
+        float: left;
+        width: 40px;
+        height: 40px;
+        border-radius: 50%;
+        margin-left: -50px;
+    }
+
+    .news-chat-quote__text-container {
+        margin-left: 50px;
+    }
+
+    .news-chat-quote__username {
+        font-weight: 600;
+        margin-bottom: 2px;
+    }
+
+    .news-chat-quote__colour-nat {
+        color: #fa3703;
+    }
+
+    .news-chat-quote__colour-no-group {
+        color: #FFFFFF;
+    }
+</style>
+
+**[You can view the match information here!](https://osu.ppy.sh/community/matches/116874748)**\
+**[You can view the match VOD here!](https://www.twitch.tv/videos/2357969267)**
+
+As the week came to a close, there was but one more surprise awaiting us in the match against ::{ flag=VE }:: Venezuela (#12) and ::{ flag=CA }:: Canada (#21)! Everyone's expectations were sky-high, as this was the first time Venezuela had the chance to go against Canada in a World Cup match, who reached Semifinals just last year. Nobody expected the roller coaster of emotions to ensue, ending in a nerve-wracking tiebreaker to decide the victor.
+
+The first map chosen was [Tedjimo yomigY - Usuva Kagerow](https://osu.ppy.sh/beatmapsets/2307475#mania/4935023) (HB1). A grueling battle of precision, only won by a fairly narrow margin of 5,360 points in favour of Venezuela managing to maintain their cool till the very end. A testament to Venezuela's resilience and perseverance right off the bat! The next three maps also ended in victory for Venezuela, who extended their lead by such large, seemingly uncontested margins. At this point, many viewers thought the match would end in a complete sweep in favour of Venezuela.
```

Comment:
the removed sentence just seemed out of place
```suggestion
The first map chosen was [Tedjimo yomigY - Usuva Kagerow](https://osu.ppy.sh/beatmapsets/2307475#mania/4935023) (HB1). A gruelling battle of precision, only won by a fairly narrow margin of 5,360 points in favour of Venezuela, managing to maintain their cool till the very end. The next three maps also ended in victory for Venezuela, who extended their lead by such large, seemingly uncontested margins. At this point, many viewers thought the match would end in a complete sweep in favour of Venezuela.
```

Reply:
Don't disagree